source/Assets/Plugins/Didomi/Scripts/Didomi.cs

```diff
@@ -359,73 +221,6 @@ public void HidePreferences()
             didomiForPlatform.HidePreferences();
         }
 
-        /// <summary>
-        /// The initial configuration of the SDK. This should be called from an Application onCreate method. (Method with disableDidomiRemoteConfig)
-        /// </summary>
-        /// <param name="apiKey">Your API key</param>
-        /// <param name="localConfigurationPath">Path to client specific config file in JSON format. If not specified,
-        /// by defalut is set as didomi_config.json in the app assets.</param>
-        /// <param name="remoteConfigurationURL">URL to client specific remote config file in JSON format</param>
-        /// <param name="providerId">Your provider ID (if any)</param>
-        /// <param name="disableDidomiRemoteConfig">Used to disable remote configuration.</param>
-        /// <param name="languageCode">Language in which the consent UI should be displayed. 
-        /// By default, the consent UI is displayed in the language configured in the device settings,
-        /// if langauge is availabe and enabled by your configuration. 
-        /// This property allows you to override the default setting and specify a language to display the UI in.
-        /// String containing the language code e.g.: "es", "fr", etc.</param>
-        public void Initialize(
```

Comment:
It seems we forgot to deprecate the 2 `Initialize` methods, should we keep them deprecated for some additional time ? Internally it can be a simple call to the non-deprecated method with the parameters object, and we can skip the tests

Also we are removing some documentation, can we double check if it is also present in the `DidomiInitializeParameters` object?

Reply:
I just updated the documentation (by using Android doc).

About keeping the `initialize` functions, I would not advise to keep them. Many removed function here were not mark as deprecated.

Reply:
`Initialize` is a very important method used by every client, I would not remove this one without warning. It is not a big deal for us and it will prevent stressing the clients.

Reply:
done